versioned_docs/version-3.21/introduction/getting-started.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Getting Started
+---
+
+Nirvana is written in C# using [.NET Core](https://www.microsoft.com/net/download/core) (an amazing runtime environment that currently runs on Windows, Linux, Mac OS X, and in Docker images). Once .NET Core has been downloaded, all you need to do is grab the source, compile it, and grab the data files.
+
+:::tip
+Nirvana currently uses .NET6.0. Please make sure that you have the most current runtime from the [.NET Core downloads](https://www.microsoft.com/net/download/core) page.
+:::
+
+## Quick Start
+
+If you want to get started right away, we've created [a script](https://illumina.github.io/NirvanaDocumentation/files/TestNirvana.sh) that downloads Nirvana, compiles it, and starts annotating a test file:
+
+```bash
+curl -O https://illumina.github.io/NirvanaDocumentation/files/TestNirvana.sh
+bash ./TestNirvana.sh
+```
+
+We have verified that this script works on Windows (using Git Bash or WSL), Linux, and Mac OS X.
+
+## Getting Nirvana
+
+### Compile from Source
+
+The following will grab the latest version of Nirvana from GitHub and compile it using the .NET Core compiler:
+
+```bash
+git clone https://github.com/Illumina/Nirvana.git
+cd Nirvana
+dotnet build -c Release
+```
+
+### GitHub Release Notes
+
+Alternatively, you can grab the latest binaries from our [GitHub Releases](https://github.com/Illumina/Nirvana/releases) page:
```

Comment:
Should we change the github link to git.illumina link since 3.21 will reside on git.illumina?